designsafe/apps/projects/models/agave/hybrid_simulation.py

```diff
@@ -88,7 +88,7 @@ def to_datacite_json(self):
         attributes['relatedIdentifiers'] = []
         for r_work in self.related_work:
             identifier = {}
-            mapping = {'Linked Project': 'IsSupplementTo', 'Cited By': 'IsCitedBy', 'Context': 'IsDocumentedBy'}
```

Comment:
May need to keep both Linked Project and Linked Dataset just in case a user has an older project where it was showing as Project, so the data goes to Datacite properly